sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -669,7 +669,9 @@ case class NthValue(input: Expression, offsetExpr: Expression, ignoreNulls: Bool
 
```

Comment:
nit: `NthValue` -> `Nth value` in the doc
Can we also add an example about `ignore nulls` while we're here?

Reply:
Unfortunately there is no SQL API for it yet. It's only available in DataFrame API. We will add examples when we add the SQL API.

Reply:
@cloud-fan, isn't it registered in SQL side? I see the example added in the original PR https://github.com/apache/spark/commit/8b09536cdf5c5477114cc11601c8b68c70408279#diff-5e6cbbb39e10b12ccd9d6a0bc3de31a4R153

Reply:
The current `ignore nulls` is very ustomized. I think it needs to be redesigned to be more common to all window functions.

Reply:
ahh gotya

Reply:
I mean the `ignore nulls` feature.